external/model-preparation-algorithm/mpa_tasks/extensions/datasets/mpa_cls_dataset.py

```diff
@@ -46,15 +47,16 @@ def __init__(self, ote_dataset=None, labels=None, **kwargs):
     def load_annotations(self):
         for dataset_item in self.ote_dataset:
             if dataset_item.get_annotations() == []:
-                label = None
+                continue
```

Comment:
Can you explain why you make it "continue"? I think gt_label can be wrong-aligned with images when calling __getitem__ with index.

Reply:
Same with above comment.